### ISSUE TITLE
S3 listener in Localstack sends version 2.0 events (latest are 2.1)

### ISSUE THREAD

**Issue by Antauri** [localstack/localstack#3595]
# Type of request: 
This is a request to update the version of the s3_listener.py to the latest 2.1 in the AWS docs or support configuring the message type?

[x] bug report
[x] feature request

# Detailed description
We were writing tests using Localstack and against version 2.1 of events (from our production). Trying to write tests for our code in Localstack we were surprised of the filtering function (specifically set at 2.1) not allowing data to pass. It seems "2.0" is hard-coded in the Localstack s3_listener.py.

Direct link: 
https://github.com/localstack/localstack/blob/master/localstack/services/s3/s3_listener.py#L159

...

## Expected behavior
Was expecting events to be generated, usually with the latest known version (this was 2.1).
...

## Actual behavior
Events are generated with 2.0 which was at the moment probably s3_listener.py in Localstack was developed.
...

# Steps to reproduce
Run in Java/Camel, something similar to:
```
.split().jsonpath("Records")
            .to("log:SqsPreFiltering?level=INFO")
            .filter(simple("${body[eventVersion]} == '2.1'"))
            .to("log:SqsPreFilteringX1?level=INFO")
            .filter(simple("${body[eventSource]} == 'aws:s3'"))
            .to("log:SqsPreFilteringX2?level=INFO")
            .filter(simple("${body[eventName]} starts with 'ObjectCreated'"))
```

## Command used to start LocalStack
testcontainers.org (as part of CI tests)
...

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
same, testcontainers.org
...

**Comment by whummer** [comment]
Thanks for reporting @Antauri . This is fixed in #3604 and should now be available in the `latest` Docker image. Thanks

**Comment by Antauri** [comment]
Thank you kind sirs!
